hack/make/validate-lint

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+source "${MAKEDIR}/.validate"
+
+# We will eventually get to the point when packages should be the complete list
+# of subpackages, vendoring excluded, as given by:
+#  go list ./... | grep -v "^github.com/docker/docker/vendor"
+packages=(
+	"builder/parser/dumper"
+	"daemon/events"
+	"daemon/execdriver/native/template"
+	"docker"
+	"dockerinit"
+	"pkg/chrootarchive"
+	"pkg/directory"
+	"pkg/fileutils"
+	"pkg/homedir"
+	"pkg/listenbuffer"
+	"pkg/mflag/example"
+	"pkg/promise"
+	"pkg/pubsub"
+	"pkg/random"
+	"pkg/symlink"
+	"pkg/timeutils"
+	"pkg/tlsconfig"
+	"pkg/urlutil"
+	"pkg/version"
```

Comment:
Would you mind either stripping the double quotes from these or swapping them to single quotes?